.gitignore

```diff
@@ -3,9 +3,14 @@
 *.exe
 *.o
 *.so
-*.pyd
-*.pyc
+*.py[cdo]
+__pycache__/
```

Comment:
UC2 is a python2 app. So there is no sense handling py3 files. pyo files are also out of handling scope because no sense running python with -O flag.

Reply:
> no sense handling py3 files
> because no sense running python with -O flag.

But somebody might try python3 or using `-O`. This is just to avoid other developers repo to be cluttered - and does no harm.

Reply:
If somebody don't read readme files before cloning code it seems this is a problem of somebody.